### PR TITLE
DIG-1388: update candig-ingest with new katsu endpoints

### PR DESCRIPTION
**Pull request by daisieh**
All tests in test-integration should still pass. (It is possible that katsu tests will fail, but test_ingest_permissions should pass)

### CODE REVIEW

File: katsu_ingest.py [hunk 1]
```diff
@@ -83,31 +97,36 @@ def ingest_schemas(fields, headers):
...
-                    return result, 200
+                    break
+                else:
+                    try:
+                        if "error" in response.json():
+                            result["errors"].append(f"{type}: {response.status_code} {response.json()['error']}")
+                    except:
+                        message = f"\nREQUEST STATUS CODE: {response.status_code} \nRETURN MESSAGE: {response.text}\n"
+                        result["errors"].append(f"{type}: {message}")
+                    if type == "programs" and "unique" in response.text:
```
Comment (OrdiNeu) [inline katsu_ingest.py:126]: I'm curious: what's this conditional checking? And why does it return instead of continuing the for loop?

**Review by OrdiNeu — COMMENTED** [review]
Unsure if it's just on this PR, but I'm getting an error on `test_ingest_permissions` where it's returning 500 instead of 403.

**Comment by daisieh** [comment]
Make sure the version of clínical_ETL_code is the one from https://github.com/CanDIG/clinical_ETL_code/pull/44?

**Comment by daisieh** [comment]
If you're still getting 500, can you post the logs from the ingest container?

**Comment by OrdiNeu** [comment]
Ah, yeah that was it

**Review by OrdiNeu — APPROVED** [review]

**Pull request merged**